external/coreclr/project.json.template

```diff
@@ -2,16 +2,16 @@
   "frameworks": {
     "{TFM}": {
       "dependencies": {
-        "Microsoft.NETCore.Platforms": "1.2.0-beta-24721-02",
-        "Microsoft.NETCore.Runtime.CoreCLR": "1.2.0-beta-24904-03",
-        "Microsoft.NETCore.TestHost": "1.2.0-beta-24904-03",
-        "runtime.native.System.Data.SqlClient.sni": "4.4.0-beta-24814-01",
+        "Microsoft.NETCore.Platforms": "2.0.0-beta-24916-02",
```

Comment:
I see, misread your original question.  I thought you were talking about eliminating the MS.NETCore.Platforms package.  
NuGet only picks up runtime.json if it appears in the root of a referenced package.  If you wanted to get rid of the consumption of this package you'd need to construct a package with this runtime.json in it and ensure that's part of the compile graph.